common/src/main/scala/explore/common/SimbadSearch.scala

```diff
@@ -22,32 +21,37 @@ import scala.concurrent.duration._
 object SimbadSearch {
   import RetryHelpers._
 
-  def search[F[_]: Async: Logger](term: NonEmptyString): F[Option[SiderealTarget]] =
+  def search[F[_]: Async: Logger](
+    term:     NonEmptyString,
+    wildcard: Boolean = false
+  ): F[List[SiderealTarget]] = {
+    val baseURL =
+      uri"https://simbad.u-strasbg.fr/simbad/sim-id"
+        .withQueryParam("Ident", term.value)
+        .withQueryParam("output.format", "VOTable")
+    val url     =
+      if (wildcard)
+        baseURL.withQueryParam("NbIdent", "wild")
+      else
+        baseURL
+
     retryingOnAllErrors(retryPolicy[F], logError[F]("Simbad")) {
       FetchClientBuilder[F]
-        .withRequestTimeout(5.seconds)
+        .withRequestTimeout(20.seconds)
```

Comment:
Will this timeout apply to all simbad queries?

Reply:
Yes. We could make it a parameter if need be. Wildcard queries tend to take a bit longer.